app/views/about/plugins_empty_state/index.json.rabl

```diff
@@ -0,0 +1,6 @@
+object false
+node do |u|
+  { :header => N_("Plugin"), :description => N_("Plugins are tools to extend and modify the functionality of Foreman. Plugins offer custom functions and features so that each user can tailor their environment to their specific needs."),
```

Comment:
this makes me wonder why not simply store this inside a component ? e.g. whats the value of making an api call to get static values?